db/migrate/20150617184118_create_tasks.rb

```diff
@@ -1,11 +1,14 @@
 class CreateTasks < ActiveRecord::Migration
   def change
     create_table :tasks do |t|
-      t.string :name, null: false
-      t.string :description
+      t.string   :name, null: false
+      t.string   :description
       t.datetime :date_complete
+      t.integer  :person_id
```

Comment:
Once a migration has been merged into master it's generally a bad idea to update it as it becomes difficult to keep other team mates databases in sync. I would suggest creating a new migration and adding this column there.